packages/framework-provider-azure/src/library/scheduled-adapter.ts

```diff
@@ -0,0 +1,26 @@
+import { Logger, ScheduledCommandEnvelope, UUID } from '@boostercloud/framework-types'
+
+interface AzureScheduledCommandEnvelope {
+  bindings: {
+    [name: string]: object
```

Comment:
I'd avoid using `object` as it is equivalent to `any`. Better to use `unknown`

```suggestion
    [name: string]: unknown
```